apps/docs/src/app/core/component-docs/combobox/examples/combobox-byline-example.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component } from '@angular/core';
+import { MobileModeConfig } from '@fundamental-ngx/core/mobile-mode';
+
+@Component({
+    selector: 'fd-combobox-byline-example',
+    templateUrl: './combobox-byline-example.component.html',
+    styleUrls: ['combobox-example.component.scss']
+})
+export class ComboboxBylineExampleComponent {
+    values = [
+        { name: 'Photo Voltaic', icon: 'photo-voltaic' },
+        { name: 'Settings', icon: 'settings' },
+        { name: 'Heating Cooling', icon: 'heating-cooling' },
+        { name: 'Competitor', icon: 'competitor' },
+        { name: 'Chalkboard', icon: 'chalkboard' },
+        { name: 'Database', icon: 'database' },
+        { name: 'Passenger Train', icon: 'passenger-train' },
+        { name: 'World', icon: 'world' },
+        { name: 'Shield', icon: 'shield' },
+        { name: 'Journey Change', icon: 'journey-change' }
+    ];
+
+    mobileConfig: MobileModeConfig = {
+        title: 'Title',
+        approveButtonText: 'Approve',
+        cancelButtonText: 'Cancel',
+        hasCloseButton: true,
+        dialogConfig: {
+            ariaLabel: 'Title'
+        }
+    };
+
+    selected: any;
+
+    selected2: any;
```

Comment:
is this being used?

Reply:
also I think we can define actual type here

Reply:
selected2 is used for mobile example